CoinNames.txt

```diff
@@ -1,4 +1,4 @@
-{
+﻿{
```

Comment:
Double check that the encoding is correct.

Reply:
ï»¿{
    "01": 

Encoding IS UTF-8 with BOM